client-js/app/scripts/listbuilder.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Allows for the generation of a large list of DOM nodes by chunking the
+ * rendering.
+ */
+export class ListBuilder {
```

Comment:
Does (<iron-list>)[https://elements.polymer-project.org/elements/iron-list] have this functionality built in already?